mleap-core/src/test/scala/ml/combust/mleap/core/feature/MaxAbsScalerModelSpec.scala

```diff
@@ -13,8 +13,7 @@ class MaxAbsScalerModelSpec extends FunSpec {
 
     it("Scales the vector based on absolute max value"){
       val inputArray = Array(15.0, -5.0, 5.0, 19.0)
-      val expectedVector = Array(0.75, -0.5, 0.5, 0.95)
-
+      val expectedVector = Array(0.75, -0.5, 0.5, 0.9500000000000001)
```

Comment:
0.9500000000000001 make test flaky ? Why not use use eq with error tolerance ?

Reply:
We need to refactor the parity check to allow tolerance. It is comparing row by row right now.